tensorstore/kvstore/tsgrpc/kvstore_server_test.cc

```diff
@@ -91,6 +94,20 @@ TEST_F(KvStoreTest, Basic) {
   tensorstore::internal::TestKeyValueReadWriteOps(store);
 }
 
+TEST_F(KvStoreTest, Multipart) {
+  auto context = tensorstore::Context::Default();
```

Comment:
Use something like this:

```
  // uses 2mb chunks; this will force at least two writes.
  tensorstore::internal::FlatCordBuilder cord_builder(16 + 2 * 1024 * 1024);
  memset(cord_builder.data(), 0x37, cord_builder.size());
  absl::Cord data = std::move(cord_builder).Build();
  data.Append("abcd");  // second chunk.
```

With these includes:

```
#include <cstring>
#include <utility>
#include "third_party/tensorstore/internal/flat_cord_builder.h"
```

Reply:
Likewise in tsgrpc_test.cc

Reply:
What's the idea behind this change? Would you like to (implicitly) fix the fact that default value of `max_sent_part_bytes` (or `max_message_bytes` as you proposed to rename) equals to 2 mib? If yes, I can alternatively introduce a separate test that ensures that default part size is 2 mib without changing much the existing tests; to me it looks like they do their job and cover all "chunking" logic.

Also, IMHO using long strings in tsgrpc_test.cc would be less elegant, as the rest of tests constructs proto messages using ParseTextProtoOrDie, which makes understanding the communication quite easy. 

But if you insist, I'll rewrite all unittests using 2 mib chunks.

Also, why do you suggest `16 + 2 * 1024 * 1024`? The first chunk would in fact be the first 2 mib, and the second would consist of 16 times (char)0x37 and abcd in your example, and not of "abcd" solely as your comment suggests.